src/app/photos/components/Preview.tsx

```diff
@@ -30,43 +30,60 @@ export default function Preview({
   const [thumbnailSrc, setThumbnailSrc] = useState<string | null>(null);
 
   useEffect(() => {
-    if (previewIndex !== null && bucketId) {
-      setThumbnailSrc(null);
-
-      const photo = items[previewIndex];
-      getPhotoPreview({
-        photo,
-        bucketId,
-      }).then(setThumbnailSrc);
-    }
+    if (previewIndex === null) return;
+    setThumbnailSrc(null);
+    const photo = items[previewIndex];
+    const photoBucketId = photo.networkBucketId ? photo.networkBucketId : bucketId;
+    if (!photoBucketId) return;
+    getPhotoPreview({
+      photo,
+      bucketId: photoBucketId,
+    })
+      .then(setThumbnailSrc)
+      .catch((err) => {
+        Sentry.captureException(err, {
+          extra: {
+            photoId: photo.id,
+            bucketId: photoBucketId,
+          },
+        });
+      });
   }, [previewIndex, items]);
 
   const [src, setSrc] = useState<string | null>(null);
   const [itemType, setItemType] = useState<PhotosItemType>(PhotosItemType.PHOTO);
 
   useEffect(() => {
-    if (previewIndex !== null && bucketId) {
-      setSrc(null);
+    if (previewIndex === null) return;
+
+    setSrc(null);
 
-      const abortController = new AbortController();
-      const photo = items[previewIndex];
-      getPhotoBlob({ photo, bucketId, abortController })
-        .then((blob) => {
-          setItemType(photo.itemType);
-          return setSrc(URL.createObjectURL(blob));
-        })
-        .catch((err) => {
-          if (abortController.signal.aborted) {
-            return;
-          }
+    const abortController = new AbortController();
+    const photo = items[previewIndex];
+    const photoBucketId = photo.networkBucketId ? photo.networkBucketId : bucketId;
```

Comment:
- networkBucketId could be undefined for Photos created before we introduced the property in photos-server
- the bucketId is a property that comes from Redux, so it can be undefined if is not initialized yet, but should exist always

Should not happen the case where both doesn't exists, at least the bucketId should has value